backend/drive/drive.go

```diff
@@ -2841,25 +2841,25 @@ func (f *Fs) makeShortcut(ctx context.Context, srcPath string, dstFs *Fs, dstPat
 	return dstFs.newObjectWithInfo(dstPath, info)
 }
 
-// List all team drives
-func (f *Fs) listTeamDrives(ctx context.Context) (drives []*drive.TeamDrive, err error) {
-	drives = []*drive.TeamDrive{}
-	listTeamDrives := f.svc.Teamdrives.List().PageSize(100)
+// List all shared drives
+func (f *Fs) listSharedDrives(ctx context.Context) (drives []*drive.SharedDrive, err error) {
+	drives = []*drive.SharedDrive{}
```

Comment:
We can't change this - I think the data structure is still called `drive.TeamDrive` not `drive.SharedDrive` see [the docs](https://godoc.org/google.golang.org/api/drive/v3).